start-tests.sh

```diff
@@ -138,6 +138,15 @@ else
   echo -e "\nExcluding tests that contain the following tags: ${YELLOW}$CYPRESS_TAGS_EXCLUDE${NC}\033[0m"
 fi
 
+if [[ $PROW_MODE == true ]]; then
+  echo -e "Checking pod status in $installNamespace:"
+  oc get pods $ADD_KUBECONFIG -n $installNamespace
+  echo -e
+
+  echo -e "Waiting for an additional 2 minutes to ensure that all pods are up and running in the cluster."
+  sleep 120
```

Comment:
I think you have had some problems with pods not coming up before, so maybe we can add some logic to confirm that everything we need is running.  Narrow it down to the pods we need to avoid failing when other components have problems.

```
SEARCH_RUNNING_COUNT=${oc get pod -n open-cluster-management | grep search- |grep Running | wc -l)
```
```
CONSOLE_RUNNING_COUNT=$(oc get pod -n open-cluster-management | grep search- |grep Running | wc -l)
```

Then log a message and maybe wait longer if the count doesn't match what we expect.

Reply:
I agree with this solution. I will create a later PR adding this as an enhancement.